Realm.xcodeproj/project.pbxproj

```diff
@@ -2567,6 +2567,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		3F1DDB5825155E33007A9630 /* Carthage post-build workaround */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
What is this?

Reply:
Xcode 14 now warns when you have a run script task which is triggered based on dependency analysis but doesn't actually declare any dependencies. This tells Xcode to just run it after every build, which is what we need for this.